README.md

```diff
@@ -23,6 +23,14 @@
 
 ## Installation
 
+### Using pip
+
+```bash
+pip install https://github.com/IntelLabs/MART/archive/refs/tags/v0.2.1.zip
```

Comment:
Shall we replace the URL with `https://github.com/IntelLabs/MART/archive/refs/heads/main.zip`? I guess we don't want to update this for every release. It will be good to give an example of installing specific version though ,maybe in comment?

Reply:
Yes, I think we can make this more generic.